rust/lance/src/io/exec/planner.rs

```diff
@@ -512,16 +512,24 @@ impl Planner {
         }
     }
 
-    /// Create Logical [Expr] from a SQL filter clause.
+    /// Create Logical [Expr] from a SQL column expression clause.
     ///
     /// Note: the returned expression must be passed through [optimize_expr()]
     /// before being passed to [create_physical_expr()].
-    pub fn parse_filter(&self, filter: &str) -> Result<Expr> {
+    pub fn parse_column_expr(&self, expr: &str) -> Result<Expr> {
```

Comment:
What's the difference between this and `parse_expr` below?

Reply:
huh didn't realize that function existed. Will fix

Reply:
fixed